### ISSUE TITLE
Questions regarding HTTP 2.0 response header

### ISSUE THREAD

**Issue by rtabor** [ambertests/charles_to_postman#1]
Hi Amber, I wasn't sure how to get in contact with you other than opening an issue.

I wanted to say your script is awesome!  Creating a script like this is something that our team has had on the back burner for a little while now.

I did run into a situation where the script is failing on a key error when processing response headers from HTTP 2.0 requests
Most of the requests made from our API are using HTTP 2.0.  I noticed that HTTP 1.1 requests include `firstLine` under `header` in the response, but HTTP 2.0 does not.

I added a simple try/except block to a local copy of the script to skip over that processing if the header is not there:
```
try:
	p_status = c_response['header']['firstLine'].replace(
		'%s %d ' % (protocol, status_code), '')

	p_response['status'] = status_code
except KeyError:
	pass
```

I wanted to get feedback from you on whether you were welcoming any contributions, or if you planned on doing maintenance for this script yourself?

Again thank you for your contribution!

**Comment by ambertests** [comment]
Hi Ronnie,

I'm so glad you are finding my little script useful! I'm happy to accept contributions if you want to submit a PR.

Thanks,
Amber

From: Ronnie T
Sent: Tuesday, September 26, 2:16 PM
Subject: [ambertests/charles_to_postman] Questions regarding HTTP 2.0 response header (#1)
To: ambertests/charles_to_postman
Cc: Subscribed


Hi Amber, I wasn't sure how to get in contact with you other than opening an issue.
I wanted to say your script is awesome! Creating a script like this is something that our team has had on the back burner for a little while now.
I did run into a situation where the script is failing on a key error when processing response headers from HTTP 2.0 requests
Most of the requests made from our API are using HTTP 2.0. I noticed that HTTP 1.1 requests include firstLine under header in the response, but HTTP 2.0 does not.
I added a simple try/except block to a local copy of the script to skip over that processing if the header is not there:
try: p_status = c_response['header']['firstLine'].replace( '%s %d ' % (protocol, status_code), '') p_response['status'] = status_code except KeyError: pass
I wanted to get feedback from you on whether you were welcoming any contributions, or if you planned on doing maintenance for this script yourself?
Again thank you for your contribution!
